cypress/e2e/navigation.cy.ts

```diff
@@ -72,4 +72,36 @@ describe('Navigation Test', () => {
     profilePage.verifySchoolDetails(schoolDetails);
     cy.logout();
   });
+
+  it('Should navigate to enrolled contests and verify filter', () => {
+    const userLoginOptions = loginPage.registerMultipleUsers(1);
+    cy.login(userLoginOptions[0]);
+
+    cy.get('[data-nav-user]').should('be.visible');
+    cy.get('[data-nav-user]').click();
+    cy.get('[data-nav-user-contests-enrolled]').should('be.visible');
+
+    cy.get('[data-nav-user-contests-enrolled]').first().click();
```

Comment:
Casi! Ya no necesitas indicar cual de los selectores quieres porque ya es único:
```suggestion
    cy.get('[data-nav-user-contests-enrolled]').click();
```